social-proof-section-master/styles.css

```diff
@@ -0,0 +1,275 @@
+html {
+  height: 100%;
+}
+
+body {
+  background-color: hsl(0, 0%, 100%);
+  color: hsl(300, 43%, 22%);
+  font-size: 15px;
+  font-family: "League Spartan";
+  font-weight: 400;
+  height: 100%;
+  line-height: 1.2;
+}
+
+h1 {
+  font-weight: 700;
+  font-size: 3em;
+  line-height: 1;
+}
+
+p {
+  line-height: 1.5;
+  font-weight: 500;
+}
```

Comment:
I'd avoid setting styles on `html`, `body` and other attributes globally unless there is no other way or you're absolutely sure that these won't be overwritten anywhere else.

I'm aware that different browsers set different default styles, e.g. default `body` styles in Edge browser:
<img width="354" alt="image" src="https://github.com/wojdanowski/exercises/assets/100123175/01eb6d45-04b2-4f8e-a6fe-14ce8ea5aa9e">
However this can be resolved by using [CSS normalize](https://github.com/necolas/normalize.css/)  or [CSS reset](https://meyerweb.com/eric/tools/css/reset/)

For `body` styles I tend to add a `<div>` as a first element after the body and with a CSS class apply styles there